### ISSUE TITLE
Species count on higher taxa in tree API 

### ISSUE THREAD

**Issue by thomasstjerne** [CatalogueOfLife/backend#527]
In the existing CoL portal theres a species count on all higher taxa in tree.
This is probably calculated once when the release is done. Is it possible to have species counts in the tree api, at least on released catalogues ?

**Comment by mdoering** [comment]
Yes, for all datasets that are static this can be done. I would then also precalculate numGenus, numSpecies, numInfraspecies & numDescendants. How to know that a dataset is immutable still needs thinking. Maybe instead of datasetType a managed or immutable flag? In that case the MANAGED datasetType is not needed anymore.

**Comment by mdoering** [comment]
duplicate of #834
